modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenProperty.java

```diff
@@ -38,6 +38,11 @@
      * may be represented as 'int', 'int32', 'Integer', etc, depending on the programming language.
      */
     public String dataType;
+    /**
+     * Alternative version of the language-specific data type when generator needs a different output in some templates.
+     * For example this is used by typescript fetch generator to output either Array (javascript) or List (immutablejs)
+     */
+    public String dataTypeAlternate;
```

Comment:
From what I understand, the only way to have another property accessible in the mustache file for models is to add it here? If there was another way, this would be great, we would not have to "polute" global codegen here...